src/app/components_ngrx/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -73,6 +73,64 @@
 </div>
 <!-- Table Inline Quick Add Ends -->
 
+<!--Quick Add for Query Tab-->
+
+<div class="f8-quickadd-query__wrapper" *ngIf="wilistview === 'wi-query-view'">
+  <input class=""
```

Comment:
Why is class attribute empty here? Or if we do not need this you can remove this.